mscore/musescore.cpp

```diff
@@ -2812,6 +2812,23 @@ void MuseScore::showElementContext(Element* el)
       debugger->setElement(el);
       }
 
+//---------------------------------------------------------
+//   reDisplayDockWidget
+//
+//   Helper function to ensure when un-docked widgets are
+//   re-displayed, they are also re-dockable
+//---------------------------------------------------------
+
+void MuseScore::reDisplayDockWidget(QDockWidget* widget, bool visible)
+{
+      if (widget->isFloating() && !widget->isVisible()) {
+            // Ensure the widget is re-dockable if it has been closed and re-opened when un-docked
```

Comment:
Also could you please add a reference to [QTBUG-69922](https://bugreports.qt.io/browse/QTBUG-69922) (something short, like `see QTBUG-69922` would be enough)? Even though this is a really good workaround it is still a workaround, and it is better not to keep it in the code if that issue gets fixed (and ensure that we keep it while this Qt issue is still present).